corehq/apps/callcenter/dbaccessors.py

```diff
@@ -0,0 +1,5 @@
+from corehq.apps.domain.dbaccessors import get_docs_in_domain_by_class
+
+
+def get_call_center_indicator_configs_by_domain(domain):
+    return get_docs_in_domain_by_class(domain, 'CallCenterIndicatorConfig')
```

Comment:
You need to pass in the actual class. I think you can go head and inline this actually rather than making it a dbaccessor, just because (1) it's calling a dbaccessor that already has a clear semantic meaning (2) dbaccessors should each have a test, and that seems like overkill in this case. Also, one of the main concrete benefits of dbaccessors is they let their implementation be refactored later; in this case any refactoring you'd do to this you'd probably want to just do to `get_docs_in_domain_by_class` instead anyway. (In fact, that function will most likely be refactored shortly as we reorganize the domain views.)